wiki/Client/Release_stream/Lazer/Feature_comparison/en.md

```diff
@@ -0,0 +1,438 @@
+
+More discussion and consideration will be needed for how much of Aspire beatmaps will be compatible going forward. For example, invisible circles might become a properly supported feature in the future.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | Yes |
+| Needs further consideration | Yes |
+
+### osu!taiko
+
+#### The playfield aspect ratio is limited
+
+The osu!taiko playfield is stretched based on window size, making the apparent AR of gameplay change in line with window size. Originally this was limited completely as it affects playability but switched to only limiting the maximum aspect ratio due to community push-back for having it "as it was on stable".
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | No |
+| Classic mod revert support | Yes |
+| Intentionally changed | Yes |
+| Needs further consideration | Maybe |
+
+#### Notes that overlap swells cannot be hit
+
+Some gimmick maps make use of notes that overlap swells.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | No |
+| Needs further consideration | Yes |
+
+#### Drumrolls do not prevent mashing
+
+In stable, drumrolls could not be hit too quickly or too slowly. This restriction has been lifted, just like in ScoreV2.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | No |
+| Needs further consideration | Yes |
+
+#### Swells give a maximum of 50 score instead of 300 for each tick
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | No? |
+| Needs further consideration | Yes |
+
+### osu!catch
+
+<!-- TODO: are there any differences... ? -->
+
+### osu!mania
+
+#### Hold note heads and tails give judgements
+
+This functions similarly to ScoreV2 in stable.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | Yes |
+| Needs further consideration | No |
+
+#### Hold note ticks give score and are responsible for combo breaks
+
+![](img/hold-note-release-combo-break.gif)
+
+In stable, releasing during a hold note body would break combo immediately, but now it only breaks when missing a hold note tick.
+
+This does allow for cheesing (a.k.a. "manipulating") by allowing wrist-jacking when a finger-jack with another finger held would be required otherwise.
+
+|  |  |
+| :-- | :-: |
+| Breaks backwards compatibility | Yes |
+| Classic mod revert support | No |
+| Intentionally changed | Yes |
+| Needs further consideration | Yes |
+
+#### Hold notes may sometimes have missing ticks
+
+This is a bug that has not yet been fixed.
```

Comment:
Again, worth reporting as a bug rather than listing here?

I've never heard of this one FWIW.

Reply:
i heard this from you lol <https://discord.com/channels/188630481301012481/188630652340404224/1032979827596210237>

can agree removing this one because i can't find an issue or discussion about such a thing.